TASVideos/Pages/GameGroups/Edit.cshtml

```diff
@@ -41,9 +41,13 @@
 				<span asp-validation-for="GameGroup.Abbreviation" class="text-danger"></span>
 			</form-group>
 			<form-group>
-				<label asp-for="GameGroup.Description"></label>
-				<textarea asp-for="GameGroup.Description" class="form-control" rows="4"></textarea>
+				<div>
```

Comment:
I don't think you need this div

Reply:
I'm not sure I do either, I just copied it from here https://github.com/TASVideos/tasvideos/blob/main/TASVideos/Pages/UserFiles/Edit.cshtml#L27C1-L27C1

Reply:
If removed there's no line break between the formatting bar and "Description"